### ISSUE TITLE
[Bug] Xray doesn't work with Fabric api

### ISSUE THREAD

**Issue by Thatrichking2** [MeteorDevelopment/meteor-client#2409]
### Describe the bug

As soon as i put fabric api into my mods folder it just looks like this :

![Screenshot (18)](https://user-images.githubusercontent.com/107333968/174163443-89ff90fd-8f9a-4df9-b295-9ccc627b5891.png)

### Steps to reproduce

```bash
Download Fabric api and Meteor client

Put both into your mods folder

Open minecraft and go into a server

Turn on xray
```

### Meteor Version

0.50 - 1505

### Minecraft Version

1.19

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by TalEdd** [comment]
![2022-06-19_00 59 02](https://user-images.githubusercontent.com/63492921/174448890-e15ecc5a-51b5-44aa-8fce-1d23e259f1f3.png)
Seems fine to me in the latest pull-request-build🤔

**Comment by WERooo** [comment]
when i had the same issue, i receive error in console.
`Mixin apply for mod meteor-client failed meteor-client-indigo.mixins.json:AbstractQuadRendererMixin from mod meteor-client -> net.fabricmc.fabric.impl.client.indigo.renderer.render.AbstractQuadRenderer: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @Inject annotation on onBufferQuad could not find any targets matching 'bufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/client/render/RenderLayer;)V' in net.fabricmc.fabric.impl.client.indigo.renderer.render.AbstractQuadRenderer. Using refmap meteor-client-refmap.json [ -> handler$bfe000$onBufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/class_1921;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @Inject annotation on onBufferQuad could not find any targets matching 'bufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/client/render/RenderLayer;)V' in net.fabricmc.fabric.impl.client.indigo.renderer.render.AbstractQuadRenderer. Using refmap meteor-client-refmap.json [ -> handler$bfe000$onBufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/class_1921;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]`
i fixed this by replacing RenderLayer to obfuscated one from target descriptor

**Comment by cakarlen** [comment]
@WERooo I'm having this issue also. Can you send over the code snippet to fix this?

**Comment by WERooo** [comment]
java/meteordevelopment/meteorclient/mixin/indigo/AbstractQuadRendererMixin.java

line 37
@Inject(method = "bufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/class_1921;)V", at = @At("HEAD"), cancellable = true, remap = false)

**Comment by MineGame159** [comment]
@WERooo @Thatrichking2 one of you please send the exact fabric loader version, fabric api version and meteor version.

**Comment by WERooo** [comment]
> @WERooo @Thatrichking2 one of you please send the exact fabric loader version, fabric api version and meteor version.

i installed an fresh instance with latest dev build from website and latest fabric api but i still getting this strange error

`Mixin apply for mod meteor-client failed meteor-client-indigo.mixins.json:AbstractQuadRendererMixin from mod meteor-client -> net.fabricmc.fabric.impl.client.indigo.renderer.render.AbstractQuadRenderer: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @Inject annotation on onBufferQuad could not find any targets matching 'bufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/client/render/RenderLayer;)V' in net.fabricmc.fabric.impl.client.indigo.renderer.render.AbstractQuadRenderer. Using refmap meteor-client-refmap.json [ -> handler$zpa000$onBufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/class_1921;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @Inject annotation on onBufferQuad could not find any targets matching 'bufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/client/render/RenderLayer;)V' in net.fabricmc.fabric.impl.client.indigo.renderer.render.AbstractQuadRenderer. Using refmap meteor-client-refmap.json [ -> handler$zpa000$onBufferQuad(Lnet/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl;Lnet/minecraft/class_1921;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]`

Loader version: 0.14.8
Mods:
fabric-api-0.56.1+1.19
meteor-client-0.5.0-1509

**Comment by MineGame159** [comment]
Ok well now I can reproduce it, something must have changed in fabric api in the last few days. Gonna investigate tomorrow.

**Comment by MineGame159** [comment]
Fixed in 74104436c75057165f14e8012bab78c61062de40.
